docker-compose.override.yml

```diff
@@ -0,0 +1,139 @@
+version: "2.4"
+
+services:
+  # db:
+  #   container_name: openimis-db
+  #   build:
+  #     context: ../openimis-db_dkr
+  #     args:
+  #       - ACCEPT_EULA=${ACCEPT_EULA}
+  #       - SA_PASSWORD=${DB_PASSWORD}
+  #       - SQL_SCRIPT_URL=${DB_SQL_SCRIPT}
+  #   networks:
+  #     - openimis-net
+    ## WARNING:
+    ## exposing the database port outside the openimis-net network
+    ## may lead to security issue (depending on your network topology)
+    # ports:
+    #  - 1433:1433
+  backend:
+    platform: linux
+    container_name: openimis-backend
+    build: ../openimis-be_py
```

Comment:
why not using github link ?

Reply:
It's because the master is not using it so when I cloned it, I got it without a GitHub context